content/guides/getstarted/hello-world-in-your-browser.md

```diff
@@ -273,48 +255,73 @@ console.log("Transaction sent with hash:", txHash);
 
 ### Run the application
 
-With the client application written, you can run the code via the same `run`
-command.
+Now that you know how the client works, you can run the code via the `run`
+command. Just type it in the playground terminal and press enter.
+
+```shell
+run
+```
 
 Once your application completes, you will see output similar to this:
 
 ```shell
+$ run
 Running client...
   client.ts:
-    My address: GkxZRRNPfaUfL9XdYVfKF3rWjMcj5md6b6mpRoWpURwP
-    My balance: 5.7254472 SOL
-    Sending transaction...
-    Transaction sent with hash: 2Ra7D9JoqeNsax9HmNq6MB4qWtKPGcLwoqQ27mPYsPFh3h8wignvKB2mWZVvdzCyTnp7CEZhfg2cEpbavib9mCcq
+    ProgramID:  C7JcX81YDaDJ9Bf8ebifLgBSqfKJxETek6qyTuPGJE1f
+    Transaction sent with hash: m7f7dszzdNshMZo5u2gRKjbyURk1tQHj7Hmeh3AbH7wUdnmDXmCJhA8cXJjYwVN7foJaLQiPYhEFTn6F5mWSeCb
+    Congratulations! Look at your transaction in the Solana Explorer:
+  https://explorer.solana.com/tx/m7f7dszzdNshMZo5u2gRKjbyURk1tQHj7Hmeh3AbH7wUdnmDXmCJhA8cXJjYwVN7foJaLQiPYhEFTn6F5mWSeCb?cluster=devnet
 ```
 
-### Get transaction logs
+You can now click on the link provided to view your transaction in the Solana
+Explorer. If you scroll to the bottom you should see `Hello, world!` in the
+**Log Messages** section of the program output. 🎉
```

Comment:
why is "log messages" bold and capitalized?

Reply:
It was like that :D Maybe because it shows that its something in the playground terminal? Maybe nice? Any other suggestiong?

Reply:
ahh that makes sense. we can just leave it then